### PR TITLE
Add scripts/tasks for the addition of new permission admins

### LINKED ISSUE #3: Add scripts and tests for new permission admin proposal

### PR DESCRIPTION
**Pull request by miguelmtzinf**
Closes #3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,7 +73,16 @@ All tests will run a fork-based test with the existing deployed versions of aave
...
 - `guardian` - the admin address of this contract with the permission to cancel ActionsSets and update the delay value
 
+## Adding a new PermissionAdmin
 
+Adding a new permission admin to the ARC protocol needs a governance proposal to be approved by the community and the `guardian`.
+
+The task `full:submit-proposal-new-permission-admin` lets you submit a proposal in this regard, providing the address of the new permission admin to add and the IPFS encoded hash of the AIP text.
+
+```
+ADMIN_ADDRESS="Your Address" IPFS_HASH="Your IPFS encoded hash" npm run main:proposal:submit-new-permission-admin
```
Comment (diego-G) [inline README.md]:
```suggestion
ADMIN_ADDRESS="Your Address" IPFS_ENCODED_HASH="Your IPFS encoded hash" npm run main:proposal:submit-new-permission-admin
```

File: package.json [hunk 2]
```diff
@@ -21,12 +21,14 @@
...
     "test:arc:full:tenderly": "npm run compile && TENDERLY=true npm run hardhat:tenderly -- test test/arc-full",
+    "test:arc:new-permission-admin": "npm run compile && MAINNET_FORK=true FORK_BLOCK_NUMBER=14132953 npm run hardhat test test/arc-list-permission-admin",
     "test:arc:keys": "npm run compile && MAINNET_FORK=true FORK_BLOCK_NUMBER=13627327 npm run hardhat test test/arc-transfer-keys",
     "kovan:timelock:deploy": "npm run compile && npm run hardhat:kovan -- full:deploy-arc-timelock --verify",
     "main:timelock:deploy": "npm run compile && npm run hardhat:main -- full:deploy-arc-timelock --verify",
     "kovan:proposal:deploy": "npm run compile && npm run hardhat:kovan -- full:deploy-proposal-payload --verify",
     "main:proposal:deploy": "npm run compile && npm run hardhat:main -- full:deploy-proposal-payload --verify",
     "main:proposal:submit": "npm run compile && npm run hardhat:main -- full:submit-proposal",
+    "main:proposal:submit-new-permission-admin": "npm run compile && npm run hardhat:main -- full:submit-proposal-new-permission-admin --admin ${ADMIN_ADDRESS} --ipfs ${IPFS_HASH}",
```
Comment (diego-G) [inline package.json]:
```suggestion
    "main:proposal:submit-new-permission-admin": "npm run compile && npm run hardhat:main -- full:submit-proposal-new-permission-admin --admin ${ADMIN_ADDRESS} --ipfs ${IPFS_ENCODED_HASH}",
```

Comment (diego-G) [inline package.json]:
```suggestion
    "main:proposal:submit-new-permission-admin": "npm run compile && npm run hardhat:main -- full:submit-proposal-new-permission-admin --admin ${ADMIN_ADDRESS} --ipfs ${IPFS_ENCODED_HASH}",
```

**Review by diego-G — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (diego-G) [reply README.md]: `IPFS_HASH` is misleading because in the AIPs both exist (IPFS hash & IPFS encoded hash).

File: package.json (hunk 2 above)
Comment (miguelmtzinf) [reply package.json]: Thanks a lot for the suggestions!

**Comment by miguelmtzinf** [comment]
Kudos to @diego-G for the suggestions!

**Pull request merged**